Yoast/wordpress-seo
# Introduce flag to prevent version bump in RC script

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* [non-user-facing] Introduces the grunt flag '--no-version-bump' to prevent the RC version to be bumped when this is not desired (still working on RC1 but having to restart the script).

## Relevant technical choices:

* n/a

## Test instructions

This PR can be tested by following these steps:

See the difference in the two commands:

* `grunt bump-rc-version --plugin-version=14.5 --type=release` - this should bump the RC version (RC1 --> RC2)
* `grunt bump-rc-version --plugin-version=14.5 --type=release --no-version-bump` - this should keep the current RC version the same.
* If the plugin-version is different from the current setting in the package.json, --no-version-bump still needs to set the RC version to 1.
 
## UI changes

## Documentation

## Quality assurance


Fixes #14535
